googleauth.gemspec

```diff
@@ -31,6 +31,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'jwt', '>= 1.4', '< 3.0'
   s.add_dependency 'memoist', '~> 0.12'
   s.add_dependency 'multi_json', '~> 1.11'
-  s.add_dependency 'os', '~> 0.9'
+  s.add_dependency 'os', '~> 1.0'
```

Comment:
Can we swap this for the more conservative `'>= 0.9', '< 2.0'` ?

Reply:
@TheRoyalTnetennba updated